packages/enclave-sdk/tests/sdk.test.ts

```diff
@@ -49,5 +49,26 @@ describe("encryptNumber", () => {
       expect(value.encryptedVote).to.be.an.instanceof(Uint8Array);
       expect(value.proof).to.be.an.instanceOf(Object)
     }, 9999999);
+
+    it("should encrypt a vecor of numbers without crashing in a node environent", async () => {
+      const buffer = await fs.readFile(
+        path.resolve(__dirname, "./fixtures/pubkey.bin"),
+      );
+      const value = await sdk.encryptVector(new BigUint64Array([1n, 2n]), Uint8Array.from(buffer));
+      expect(value).to.be.an.instanceof(Uint8Array);
+      expect(value.length).to.equal(27_674);
+    });
+
+    it("should encrypt a vector and generate a proof without crashing in a node environent", async () => {
+      const buffer = await fs.readFile(
+        path.resolve(__dirname, "./fixtures/pubkey.bin"),
+      );
+
+      const value = await sdk.encryptVectorAndGenProof(new BigUint64Array([1n, 2n]), Uint8Array.from(buffer), demoCircuit as unknown as CompiledCircuit);
+      
+      expect(value).to.be.an.instanceof(Object);
+      expect(value.encryptedVote).to.be.an.instanceof(Uint8Array);
```

Comment:
Should we deprecate this field and associated rust fields in favor of `encryptedData` or just `encrypted`?